### ISSUE TITLE
A note on blank cells should be provided on a sheet by sheet basis

### ISSUE THREAD

**Issue by matt-dray** [co-analysis/a11ytables#62]
We should probably insert a line above each statistical table that explains the meaning of blank cells _for that table_.

At the moment it's expected that the user will explain on the cover page what blank cells mean, but it's acceptable for blank cells to have a different meaning on a sheet by sheet basis. Users may also forget to do this, though new_a11ytable() warns when there's an empty cell in a statistical table.

It'll need to be an argument to `new_a11ytable()`, like `blank_cells`. Should the user provide a full sentence, or just the completion of the phrase 'Blank cells in this table indicate...'? Probably the former, at first.
